pkg/roachprod/roachprod.go

```diff
@@ -849,21 +854,27 @@ func Get(l *logger.Logger, clusterName, src, dest string) error {
 	return c.Get(l, c.Nodes, src, dest)
 }
 
+type PGURLOptions struct {
+	Secure     bool
+	External   bool
+	TenantName string
```

Comment:
nit: could you add a comment or a constructor which makes it clear that by default, the TenantName is the system tenant?